modules/ROOT/pages/access-client-app-id-task.adoc

```diff
@@ -1,17 +1,22 @@
-= Getting Client ID and Client Secret of a registered Client Application
+= Obtaining Client Credentials of a Registered Client Application
 ifndef::env-site,env-github[]
 include::_attributes.adoc[]
 endif::[]
 
-As a client application owner of a registered client, you might be required to provide client application credentials when accessing an API. After a contract is created between the client application and the API, you can access the API using these credentials based on the policies that are applied. For example, if the API owner has applied the Client ID Enforcement policy on the API, you must provide the client ID, and optionally, the client secret information for the client application. 
+As an owner of a xref:exchange::to-request-access.adoc[registered client application], you might be required to provide the client ID and the client secret of the application when accessing an API. After a contract is created between the client application and the API, you can access the API using these credentials based on the policies that are applied. For example, if the API owner has applied the Client ID Enforcement policy on the API, you must provide the client ID, and optionally, the client secret information for the client application. 
 
 You must have the Organization Administrator privileges of the master organization, or must be the owner of the client application to access the credentials.  
 
 To obtain the credentials of a client application:
 
 . From the left navigation pane in Anypoint Platform, click *Management Center* > *API Manager*.
-. On the left navigation, Click *Client Applications*.
+. On the left navigation in API Manager, Click *Client Applications*.
 . From the details pane, click the application for which you want to view the client credentials.
 +
-The Client ID and Client Secret credentials appear on the right for the selected application. 
+The Client ID and Client Secret credentials for the selected application appear on the right of the window. 
```

Comment:
Let's be consistent with capitalization for client ID and client secret--only upper case if you're referring to the UI. And if you're referring to the UI, use bold.  I think it's fine to just lower-case them here.

Reply:
Done! Thanks!